docs/index.src.html

```diff
@@ -2763,7 +2763,9 @@ <h1 id="api-humandiff">Difference for Humans</h1>
     <p>You can also change the locale of the string using <code>$date->locale('fr')</code> before the
         diffForHumans() call. See the <a href="#api-localization">localization</a> section for more detail.</p>
 
-    <p id="diff-for-humans-options">Since 2.9.0 diffForHumans() parameters can be passed as an array:</p>
+    <h3 id="api-humandiff-parameters">diffForHumans() parameters</h3>

```

Comment:
ID must remain because we have a link `href="#diff-for-humans-options"` also the whole URL `https://carbon.nesbot.com/docs/#diff-for-humans-options` has been shared multiple times and we don't have a particular reason to kill those links so we should keep the anchor.